pkgs/build-support/dotnet/make-nuget-source/default.nix

```
@@ -10,21 +10,21 @@ let
     inherit name;
 
     meta.description = description;
-    nativeBuildInputs = [ dotnetPackages.Nuget xml2 ];
+    nativeBuildInputs = [ python3.pythonForBuild ];
```

```suggestion
    nativeBuildInputs = [ python3 ];
```
isn't this already choosing the right one?

---

Yep